### ISSUE TITLE
Hash rate halved and then halved again 

### ISSUE THREAD

**Issue by Klezkk** [todxx/teamredminer#281]
Hash rate dropped from 50 to 36 to 17 over the course of 36 hours on my 5700xt with video output connected (just this card in the system) on Win 10 while mining eth. Changed drivers (even rolled back by two major releases), changed power scheme, disabled monitor standby. Still no improvement. Switched to lolminer, no trouble whatsoever. I do all my underclocking and tuning via VBIOS and wattman, so TRM is vanilla.

**Comment by DoruSonic** [comment]
Could you please share your bat script please? I'm using a rx 5700xt bios modded with wattman just like you and running great. I've added the following to the default script "--eth_config=B --enable_compute"

**Comment by Klezkk** [comment]
My script is the standard one. Should I edit? Also, I found out that I also get driver timeouts on the lastest WHQL even on Lolminer. Still no troubles on lolminer on previous WHQL 20.11.2.

**Comment by DoruSonic** [comment]
Try to add "--eth_config=B --enable_compute" at the end of the script just to see if it makes a difference.
Have you run the "windows_tdr_fix"?
Are you monitoring the temps and see if the gpu is not thermal throttling?

**Comment by Klezkk** [comment]
GPU is running at 65°, memory  was running super hot, but now i fixed the fan at 45% and temps are 84-86°. I run the TDR, will now add those parameters, TY

**Comment by DoruSonic** [comment]
GPU at 65 seems very high. Are this TEdge or TJct temps? If the latter they are ok. The most important temps are for the TMem and they should be below 90ºC, ideally close or below 80 (I'm running at 80-82)
If that doesn't work, you did say that you didn't have this problem using other miners correct?

**Comment by Klezkk** [comment]
Yes I was talking about TJct. Tmem is anywhere between 82 and 90 (90 when overclocked). I'm waiting for new thermal pads to install. 

_If that doesn't work, you did say that you didn't have this problem using other miners correct?_
Yes, but on driver version WHQL  21.3.1

**Comment by DoruSonic** [comment]
If you use the 21.3.1 on TRM does it work or only on other miners? Note that the 21.3.1 is the WHQL (recommended) while the current 21.3.2 is like a pre-release version so its somehow normal for it to have some bugs or even underperform in some situations. I'm using the 21.3.1 no problems

**Comment by munzuradam** [comment]
I have a similar problem with my 5600xt. Since yesterday I am getting ~30mh/s where I should be getting ~43mh/s. I haven't changed any settings, driver isn't updated, heat isn't an issue. Don't even know what's going on?
